lambdas/es/indexer/index.py

```diff
@@ -185,7 +194,8 @@ def handler(event, _):
                     elif eventname == 'ObjectCreated:Put':
                         event_type = 'Create'
                     else:
-                        event_type = eventname
+                        # we should only send either Create or Delete events
+                        return
```

Comment:
Event handling is kind of weird in the indexer before your change, but we need to handle many more event types than what are explicitly mentioned. https://docs.aws.amazon.com/AmazonS3/latest/dev/NotificationHowTo.html#notification-how-to-event-types-and-destinations

Reply:
Returning without handling these events will lead to objects not being indexed correctly